packages/net-misc/turbovnc/files/turbovnc-fix-libressl.patch

```diff
@@ -0,0 +1,21 @@
+Upstream: unknown
+Source: Void Linux,
+https://github.com/void-linux/void-packages/blob/master/srcpkgs/turbovnc/patches/fix-libressl.patch
+Reason: Fix build with LibreSSL
+
+diff --git a/unix/Xvnc/programs/Xserver/hw/vnc/rfb.h b/unix/Xvnc/programs/Xserver/hw/vnc/rfb.h
+index 9b82d2f..8c35061 100644
+--- unix/Xvnc/programs/Xserver/hw/vnc/rfb.h
++++ unix/Xvnc/programs/Xserver/hw/vnc/rfb.h
+@@ -79,6 +79,11 @@
+    the CPU count */
+ #define MAX_ENCODING_THREADS 8
+ 
++/* Fix for libressl */
```

Comment:
I have doubts about this patch.

If you want to put it somewhere then you probably should put it closer to [code where it is being used](https://github.com/TurboVNC/turbovnc/search?q=OPENSSL_INIT_SETTINGS&unscoped_q=OPENSSL_INIT_SETTINGS).

It feels like [affected code is unused unless `TVNC_USETLS` turned on](https://github.com/TurboVNC/turbovnc/blob/2.1.2/unix/Xvnc/programs/Xserver/hw/vnc/CMakeLists.txt#L26-L61). If it is auto-magically enabled for you we should probably add option or disable it or explicitly add dependency (`build+suggestion` or `build+run` depending on how hard it is linked).

I also don't see any upstream pull request for adding libressl compatibilities. In Exherbo we try to ensure that upstream gets patches. We may want to host patches if upstream is not willing to work on that (like NVidia) or have no time to reach that PR.